client/signup/steps/site-or-domain/index.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import Card from 'components/card';
+
+export default class StepOrDomain extends Component {
+	getChoices() {
+		return [
+			{
+				type: 'page',
+				label: 'Start a new site',
+				image: (
+					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 310 230">
```

Comment:
Good catch. Apparently, these are duplicated between `DesignType` and `DesignTypeWithStore` already. I added a commit to import them from the latter here, but I think we should refactor all three components to reduce redundancy in a separate PR. I'll open an issue for this when this is merged.